platforms/hyperledger-indy/images/indy-key-mgmt/README.md

```diff
@@ -1,6 +1,6 @@
 # Blockchain Automation Framework Indy indy-key-mgmt Docker image
 
-Docker image for indy key management, which allows using commands in bash.
+Docker image for indy key management, which generates identity crypto generation into Vault or on terminal output a in json format.
```

Comment:
Docker image for indy key management, which generates identity crypto and stores it into Vault or displays it onto the terminal in json format.